src/simd/rand_impl.rs

```diff
@@ -200,3 +200,91 @@ impl_rand_packed_simd!(
     packed_simd::msizex4, _0, _1, _2, _3;
     packed_simd::msizex8, _0, _1, _2, _3, _4, _5, _6, _7;
 );
+
+#[cfg(feature = "portable_simd")]
+macro_rules! impl_rand_portable_simd(
+    ($($wrapped: ty, $($i: ident),*;)*) => ($(
+        impl rand::distributions::Distribution<$wrapped> for rand::distributions::Standard {
+            #[inline(always)]
+            fn sample<R: rand::Rng + ?Sized>(&self, rng: &mut R) -> $wrapped {
+                <$wrapped>::new($(
+                    ignore_snd!([self.sample(rng)], [$i])
```

Comment:
There's a nicer specialization for `packed_simd` vectors in the `rand` crate, not sure if it's faster or by how much. It uses fill_bytes loaded into an int vector and then shuffled to fit into a float vector.